Simperium/src/main/java/com/simperium/android/CredentialsActivity.java

```diff
@@ -311,19 +311,24 @@ private void hideDialogProgress() {
         }
     }
 
+    private boolean isEmailPasswordMatch() {
```

Comment:
Should this be getting checked in the `setButtonState()` method? The sign up button is enabled for me when I put the same email address in the password field.